src/lib/client.js

```diff
@@ -98,10 +101,12 @@ class Client extends EventEmitter {
       }
 
       const quoteQuery = {
-        source_ledger: plugin.id,
+        source_address: (yield plugin.getPrefix()),
```

Comment:
Why do you get the prefix from the plugin if the core client is configured with it?